homeassistant/components/playstation_network/manifest.json

```
@@ -0,0 +1,14 @@
+{
+  "domain": "playstation_network",
+  "name": "Playstation Network",
+  "codeowners": ["@jackjpowell"],
+  "config_flow": true,
+  "dependencies": ["network"],
+  "documentation": "https://www.home-assistant.io/integrations/playstation_network",
+  "homekit": {},
+  "integration_type": "device",
```

```suggestion
  "integration_type": "service",
```

remove empty lines, documentation not needed for core integrations

---

{
  "domain": "playstation_network",
  "name": "Playstation Network",
  "codeowners": ["@jackjpowell"],
  "config_flow": true,
  "integration_type": "service",
  "iot_class": "cloud_polling",
  "requirements": ["PSNAWP-HA==2.2.1"]
}

---

I couldn't commit without documentation specified. hassfest was failing:

Integration playstation_network:
* [ERROR] [MANIFEST] Invalid manifest: required key not provided @ data['documentation']. Got None

---

oops, sorry, my bad, I was wrong 🙈